mqtt-connect/websocket/src/main/java/com/cumulocity/mqtt/connect/client/websocket/TokenSupplier.java

```diff
@@ -16,7 +17,7 @@ class TokenSupplier {
     private final TokenApi tokenApi;
     private final String topic;
     private final String subscriber;
-    private final Supplier<String> memoizedTokenSupplier;
+    private final Supplier<Token> memoizedTokenSupplier;
 
     TokenSupplier(TokenApi tokenApi, String topic, String subscriber) {
```

Comment:
Maybe rename whole class as this is not simple supplier anymore

Reply:
TokenService?